src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/RunKeyVaultTests.ps1

```diff
@@ -211,6 +211,12 @@ function Run-AllControlPlaneTests
 
     # Set-AzureRmKeyVaultAccessPolicy & Remove-AzureRmKeyVaultAccessPolicy tests.
     Run-TestProtected { Run-VaultTest { Test_SetRemoveAccessPolicyByUPN } "Test_SetRemoveAccessPolicyByUPN" } "Test_SetRemoveAccessPolicyByUPN"
+
+    <#
+    This test is disabled because it requires a user with an email address that matches their UPN.
+    Run-TestProtected { Run-VaultTest { Test_SetRemoveAccessPolicyByEmail } "Test_SetRemoveAccessPolicyByEmail" } "Test_SetRemoveAccessPolicyByEmail"
```

Comment:
These tests are currently not run under the recording framework.  I am still investigating why that is, and what I can do about it.  As far as I can tell these tests are run manually by devs before checkin.  Since they are not recorded, if this test was enabled by default, the developer would need to ensure that they have a test environment that has a user with an email address, and further, that the email address matched their UPN.  That's not easy to set up if you don't already have one.  Setting up a user with an email address requires having a user licensed for Exchange, or doing manual REST calls to AAD graph.

If these tests were being run under the recording framework then only one person would need to have the necessary test setup and all other runs would be on recorded data.  Alternatively, we could set up a permanent test environment with users that have the necessary pre-reqs.  It's also possible to automate this kind of user creation in our current, manual, test infrastructure. 

1. The right long term answer is to get the tests running in the recording framework.  However, it is non-trivial to do so.  I need some time to investigate what it takes to get us completely back in to the recording framework.  I'd rather not invest significant effort in a temporary solution.  I plan on looking in to fixing this the right way in Calcium.
2. Without this fix it's possible for a user creating a new vault to inadvertently ACL it for someone other than the person running the CmdLet.  While we show the policy created by the CmdLet as part of the output, if the person running it isn't paying attention they may end up with a policy that grants access to the vault that they didn't intend.  This should be a very rare occurrence, but I would still like to get this fix out soon.  Blocking the release of this fix on test infrastructure investments could significantly delay it.

The risk of having this test disabled is small.  The code that does the lookup by email is factored out in to it's own method that is not shared by any other code path.

I may have missed something, and of course I am open to other suggestions.